### PR TITLE
📖 Rename Dev Channel to Experimental Channel in docs and comments

### PR DESCRIPTION
**Pull request by rcebulko**

### CODE REVIEW

File: contributing/release-schedule.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 - [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
 - [Release Cadence](#release-cadence)
   - [Detailed schedule](#detailed-schedule)
-  - [AMP Dev Channel (Experimental)](#amp-dev-channel-experimental)
-  - [AMP Beta Channel (Beta)](#amp-beta-channel-beta)
+  - [AMP Experimental Channel](#amp-experimental-channel)
+  - [AMP Beta Channel](#amp-beta-channel)
```
Comment (rcebulko) [inline contributing/release-schedule.md]: Note: @mrjoro suggested merging these sections since they are rather similar and differ only in the release provided. @danielrozenberg WDYT?

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> pushed to experimental channel (<CL submit time>)
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: While you're at it, I think you can replace the word "pushed" with the word "promoted", since that's the verb we use elsewhere. WDYT?

File: contributing/release-schedule.md (hunk 1 above)
Comment (rsimha) [reply contributing/release-schedule.md]: No objection from me on this one.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 2 above)
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Do you want to make this explicit and say "... to experimental and beta channels"?

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Note: remove the backticks (``) from the link.
...
 - [ ] Release <RELEASE_NUMBER> pushed to 1% (<CL submit time>)
 - [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: You'd eventually have to replace "production" with "stable" (can be in another PR, or roped into this one if it's straightforward).

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 2 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Donezo

Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done! Also updated similar cases elsewhere in these issue templates.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 3 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: contributing/release-schedule.md (hunk 1 above)
Comment (rcebulko) [reply contributing/release-schedule.md]: Done.

**Comment by rcebulko** [comment]
I also reorganized the sections a bit. I felt the opt-in channels belong in the "Release Channels" section, and "Release Freezes" is now under "Release Cadence". WDYT?

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 4]
```diff
@@ -23,30 +23,30 @@ with the release so please keep this issue up to date:
...
 
 Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to 1% (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta Channels (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to 1% (<CL submit time>)
```
Comment (danielrozenberg) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: How about:
```markdown
- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (opt-in) channels (<CL submit time>)
- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (1% traffic) channels (<CL submit time>)
```

/cc @rsimha @mrjoro 

(nit: :%s/Channels/channels/g)

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 5]
```diff
@@ -23,30 +23,30 @@ with the release so please keep this issue up to date:
...
+- [ ] Release <RELEASE_NUMBER> promoted to Stable (<CL submit time>)
```
Comment (danielrozenberg) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Stable channel

File: .github/ISSUE_TEMPLATE/release_issue_template.md [hunk 6]
```diff
@@ -4,7 +4,7 @@
 Note to onduty:
 
 Use this issue to track a release from the initial canary release build through
-production. The community uses this issue to keep track of what is going on
+promotion to Stable. The community uses this issue to keep track of what is going on
```
Comment (danielrozenberg) [inline .github/ISSUE_TEMPLATE/release_issue_template.md]: Why do we have this template twice? 🙃

File: examples/analytics-notification-with-geo.amp.html [hunk 7]
```diff
@@ -129,6 +129,6 @@
   <h2>Notification with amp-geo demo</h2>
   <p>This page has three "amp-user-notification" elements.  They
   are configured to trigger in different countries.  One for NAFTA (CA, MX, US).  One for multiple country groups and one for the negative matched country groups.</p>
-  <p>If you are using locahost or in the dev channel you can set your country using #amp-geo= with the two letter ISO country code.  For example #amp-geo=us.</p>
+  <p>If you are using locahost or in the experimental channel you can set your country using #amp-geo= with the two letter ISO country code.  For example #amp-geo=us.</p>
```
Comment (danielrozenberg) [inline examples/analytics-notification-with-geo.amp.html]: localhost

File: .github/ISSUE_TEMPLATE/release_issue_template.md (hunk 6 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release_issue_template.md]: I don't know but I've been making sure to keep them in sync :man_shrugging:

File: examples/analytics-notification-with-geo.amp.html (hunk 7 above)
Comment (rcebulko) [reply examples/analytics-notification-with-geo.amp.html]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 5 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 4 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ assignees: ''
 Note to onduty:
 
 Use this issue to track a release from the initial canary release build through
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Rename
> canary release

to

> experimental / beta release

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ assignees: ''
...
-production. The community uses this issue to keep track of what is going on
+promotion to Stable. The community uses this issue to keep track of what is going on
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: For clarity, change to
> through promotion to stable and LTS.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 10]
```diff
@@ -23,30 +23,30 @@ with the release so please keep this issue up to date:
   release number to the GitHub tag page the first time a given release number
   appears in the checkboxes.
 - Add any updates that may be of interest to the community (such as delays) as
-  comments on this issue, including after the release is pushed to production.
+  comments on this issue, including after the release is promoted to Stable.
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: > promoted to Stable (and in some cases, LTS).

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 11]
```diff
@@ -23,30 +23,30 @@ with the release so please keep this issue up to date:
...
 Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to 1% (<CL submit time>)
-- [ ] Release <RELEASE_NUMBER> pushed to production (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (opt-in) channels (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Experimental and Beta (1% traffic) channels (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> promoted to Stable channel(<CL submit time>)
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md]: nit: For consistency, add a space before the open parenthesis.

File: .github/ISSUE_TEMPLATE/release_issue_template.md (hunk 6 above)
Comment (rsimha) [reply .github/ISSUE_TEMPLATE/release_issue_template.md]: Good catch by @danielrozenberg. The file `.github/ISSUE_TEMPLATE/release_issue_template.md` is old, and can be deleted.

Comment (rsimha) [reply .github/ISSUE_TEMPLATE/release_issue_template.md]: To be 100% sure, here are screenshots that confirms that `.github/ISSUE_TEMPLATE/release-tracking-issue.md` is the file used as a template, while `.github/ISSUE_TEMPLATE/release_issue_template.md` is not.

image
image

In addition, the [file history](https://github.com/ampproject/amphtml/commits/master/.github/ISSUE_TEMPLATE/release_issue_template.md) for `.github/ISSUE_TEMPLATE/release_issue_template.md` confirms that it hasn't been touched in almost a year.

Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release_issue_template.md]: Great find! Removed.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 8 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 9 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 10 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 11 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Nice catch; done.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 9 above)
Comment (rsimha) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: I don't see this change. Is this PR missing a commit?

File: contributing/release-schedule.md [hunk 12]
```diff
@@ -43,70 +42,57 @@ You can determine what changes are in a given build using one of the following:
...
+In determining how often we should push new versions of AMP to everyone, we have to weigh many factors including:
 
-Similar to the _Dev Channel_, when you opt into the _AMP Beta Channel_ you are only affecting the AMP JS libraries in your browser; there is no way to force visitors to your site to use the _AMP Beta Channel_ version of AMP.
+- stability for the millions of sites/billions of pages built using AMP
+- cache busting that might happen when we push a new version
+- the desire to get new features out quickly
 
-To opt your browser into the _AMP Beta Channel_, go to [the AMP experiments page](https://cdn.ampproject.org/experiments.html) and activate the "AMP RC Channel" experiment. Please subscribe to our [low-volume announcements](https://groups.google.com/forum/#!forum/amphtml-announce) mailing list to get notified about important/breaking changes about AMP.
+After considering all of these factors we have arrived at the 1-2 week push cycle. Thus far we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.
```
Comment (rsimha) [inline contributing/release-schedule.md]: Add some commas for clarity.

> After considering all of these factors, we have arrived at the 1-2 week push cycle. Thus far, we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 9 above)
Comment (rcebulko) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Odd; must have lost it during a rebase. Added anew

File: contributing/release-schedule.md (hunk 12 above)
Comment (rcebulko) [reply contributing/release-schedule.md]: Doen

File: contributing/release-schedule.md [hunk 13]
```diff
@@ -44,7 +44,7 @@ You can determine what changes are in a given build using one of the following:
 
 ### AMP Experimental and Beta Channels
 
-The _AMP Experimental Channel_ is a way to opt a browser into using the **experimental** release build of the AMP JS libraries. The Experimental Channel_ **may be less stable** and it may contain features not yet available to all users.
+The _AMP Experimental Channel_ is a way to opt a browser into using the **experimental** release build of the AMP JS libraries. The Experimental Channel\_ **may be less stable** and it may contain features not yet available to all users.
```
Comment (rsimha) [inline contributing/release-schedule.md]: The underscore after "The Experimental Channel" should be deleted.

Comment (rcebulko) [reply contributing/release-schedule.md]: The preceding underscore was missing (it's supposed to be italicized), probably accidentally deleted during a round of editing. Re-added
